src/pushInfo/style.ts

```diff
@@ -47,13 +47,18 @@ export default css`
   }
 
   .kirk-pushInfo-icon {
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
+    position: relative;
     border-radius: 50%;
     min-width: 40px;
     min-height: 40px;
     margin-right: ${space.l};
     background-color: ${color.white};
   }
+
+  :global(.kirk-pushInfo-icon .kirk-icon) {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
I don't understand the change. Why do you need a transform ?

Reply:
It's only a partial support, many bugs can be encountered, like this one

Reply:
Yes, although actually this one is due to the use of "min-height", which I can just turn to "height". Since there's only one case of PushInfo and it's always the same icon for now.

Reply:
Alright I went for the width/height instead of min-width/min-height. Less changes win.